orthopairs/README.md

```diff
@@ -0,0 +1,54 @@
+<h2> Orthopairs: Protein ortholog file generation</h2>
+
+To see how to run this script, check out the <a href="https://github.com/reactome/data-release-pipeline/new/feature/orthopairs/orthopairs#-running-orthopairs-">Running Orthopairs</a> section.
+
+This step has been rewritten in Java and completely overhauled from the Perl <a href="https://github.com/reactome/Release/tree/master/scripts/release/orthopairs">version</a>.
```

Comment:
Markdown tip: You don't need to use HTML anchor tags, you can use markdown notation. See: https://guides.github.com/features/mastering-markdown/#syntax - look for "Links".